build.sbt

```diff
@@ -5,6 +5,7 @@ val specs2V = "3.9.1"
 val slf4jV = "1.7.12"
 
 val json4sNative = "org.json4s" %% "json4s-native" % json4sV
+val json4sjackson = "org.json4s" %% "json4s-jackson" % json4sV
```

Comment:
Do we need this for some specific reason? Otherwise we could just as well import org.json4s.native.JsonMethods instead of org.json4s.jackson.JsonMethods, and get rid of one dependency.